scss/modules/_filters.scss

```diff
@@ -20,25 +20,34 @@
 //
 // Styleguide modules.filters
 
+// by default show filters offscreen
 .filters {
   @include clearfix();
   @include u-bg--neutral();
+  @include transition(left 1s ease-in-out);
+  position: absolute;
   bottom: 0;
-  height: 0;
-  top: 0;
+  height: auto;
+  top: -20px;
```

Comment:
This should be `rem` because the padding of the element that it's compensating for is set in `rem` (and it's the convention we've been using on the project).